tests/foreman/api/test_operatingsystem.py

```diff
@@ -103,19 +103,21 @@ def test_positive_create_with_name(self):
 
     @tier1
     @run_only_on('sat')
+    @skip_if_bug_open('bugzilla', 1709683)
     def test_positive_create_with_os_family(self):
         """Create operating system with every OS family possible
 
         :id: 6ad32d22-53cc-4bab-ac10-f466f75d7cc6
 
         :expectedresults: Operating system entity is created and has proper OS
             family assigned
-
+        :CaseAutomation: Automated
         :CaseImportance: Critical
         """
         for os_family in OPERATING_SYSTEMS:
             with self.subTest(os_family):
-                os = entities.OperatingSystem(family=os_family).create()
+                os = entities.OperatingSystem(
+                    family=os_family).create()
```

Comment:
```suggestion
                    os = entities.OperatingSystem(family=os_family).create()
```
please correct above ..

Reply:
Done.